website/docs/concepts/html/components.mdx

```diff
@@ -33,48 +23,32 @@ struct Props {
     prop2: String,
```

Comment:
Maybe we should promote the `AttrValue` type here?

I didn't find any documentation about `AttrValue` so this can be solved in a future pull request as well.

Reply:
I think just use a better naming instead of `prop1` and `prop2` is sufficient as to not overload this example with new information. Fully agree with documenting `AttrValue` somewhere though.

Reply:
I'd prefer using `AttrValue`. From the call-site, there shouldn't be any difference. We can/should document it somewhere too.

Reply:
Lets address this with a separate PR.